client/components/checklist/style.scss

```diff
@@ -2,20 +2,38 @@
 	display: flex;
 	flex-direction: row;
 
+	@include breakpoint( '<480px' ) {
+		flex-direction: column;
+		justify-content: flex-end;
+	}
+
+	&.card {
+		padding: 0;
+	}
+
 	&-main {
 		display: flex;
 		flex: 1 1;
 		flex-direction: column;
 		align-items: stretch;
+		padding: 16px 24px;
+		@include breakpoint( '<480px' ) {
+			padding: 16px;
+		}
 	}
 
 	&-secondary {
+		align-items: stretch;
```

Comment:
This caused a small change that I'll need to address:

before
<img width="613" alt="Screenshot 2019-05-17 at 13 26 49" src="https://user-images.githubusercontent.com/87168/57922026-7b004e80-78a7-11e9-8ebf-78105b7fde70.png">

vs now the text is not centered vertically anymore

<img width="576" alt="Screenshot 2019-05-17 at 13 25 44" src="https://user-images.githubusercontent.com/87168/57922004-68861500-78a7-11e9-91ea-aa37cc0dff32.png">

Reply:
Seems to fix:

```css
.checklist__header-action {
  align-self: center;
}
```

Reply:
https://github.com/Automattic/wp-calypso/pull/33125/commits/e97f9b09d4788f7fc9fb640082e187161b77c7ad